### DISCUSSION TITLE
Which city currently has the highest population by urban area in the world?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#150] [Galaxy Trivia] [answered]
**Hint:** 🗼

**Comment by YuetianW** [comment, marked as answer]
According to Wikipedia, the city with the highest population by urban area is Tokyo, Japan. Urban area is a loose term referring to a contiguous area with a certain population density.
Ref：https://en.wikipedia.org/wiki/List_of_largest_cities
